doc/core/combiner-explainer.md

```diff
@@ -114,40 +114,30 @@ if we know that it is valid to do so:
 while (q.pop(&f)) {
   f();
   if (control_can_be_returned && some_still_queued_thing_is_covered_by_poller) {
-    offload_combiner_work_to_some_other_thread();
+    queue_offload(); // Queue offload work to some other thread
   }
 }
 ```
 
-`offload` is more than `break`; it does `break` but also causes some
-other thread that is currently waiting on a poll to break out of its
-poll. This is done by setting up a per-polling-island work-queue
-(distributor) wakeup FD. The work-queue is the converse of the combiner; it
-tries to spray events onto as many threads as possible to get as much concurrency as possible.
+`queue_offload` detaches the combiner from the current thread's executor and
+ offloads to the work to `executor` which is an internal pool of threads.
```

Comment:
offloads which work?

Reply:
Also, should probably be "offloads the work to " rather than "offloads _to_ the work to"

Reply:
Should I phrase it as _"offloads the remaining work to the `executor`"_  ? or is the term "work" ambiguous?

If so, I can rewrite it the following way:

_"Offloads the remaining list of closures (that were scheduled on this combiner) to the `executor`"_